packages/codemod/src/codemods/lib/add-comment.ts

```diff
@@ -0,0 +1,43 @@
+import type { API, ASTPath } from 'jscodeshift';
+
+export const AI_SDK_CODEMOD_ERROR_PREFIX = 'FIXME(@ai-sdk-codemod-error): ';
```

Comment:
why codemode-error? I think I'd make more clear that this is related to the v5 upgrade. I wouldn't call it error since the codemod is working as expected by adding that comment

```suggestion
export const AI_SDK_CODEMOD_ERROR_PREFIX = 'FIXME(@ai-sdk-upgrade-v5): ';
```

Reply:
I was following nextjs msg but yeah error doesnt make much sense here, will update to this